prime-router/src/test/resources/fhirengine/smoketest/valid_hl7.hl7

```diff
@@ -1,6 +1,6 @@
 MSH|^~\&#|MEDITECH^2.16.840.1.114222.4.3.2.2.1.321.111^ISO|COCAA^^ISO|DCEED.ELR^2.16.840.1.114222.4.1.144.2.5^ISO|CDPHE^2.16.840.1.114222.4.1.144^ISO|20280808092805-0600||ORU^R01^ORU_R01|MT_COCAA_ORU_AAPHELR.1.6214638|P|2.5.1|||||||||PHLabReport-NoAck^HL7^2.16.840.1.113883.1.11^ISO
 SFT|MEDITECH, Inc.^L^^^^MEDITECH&1.3.6.1.4.1.24310&ISO^XX^^^MEDITECH|5.67|Laboratory Applicati|20210810||20210810
-PID|1||100001^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^MR^COCAA&&ISO~285-64-4175^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^SS^COCAA&&ISO~X605236^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^PI^COCAA&&ISO~E40014904640^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^AN^COCAA&&ISO||BIERLY^LONNIE^^^^^L||19361023|M||2106-3^White^HL70005^W^White^L^2.5.1^5.67|4673 ELM ST^CHERRY CRK NURSING CTR^CAPITAN^IG^80014^USA^H||^PRN^PH^^1^303^6930111|^WPN^PH^^1^303^9999999^^Business||S|NOE^Nonreligious^HL70006|||||N^Not Hispanic or Latino^HL70189^2^Not Hispanic or Latin^L^2.5.1^5.61||||||||N|||20210809085200-0600|COCAA^^ISO|337915000^Homo sapiens (organism)^SCT^Human^Human^L^1^1
+PID|1||100001^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^MR^COCAA&&ISO~285-64-4175^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^SS^COCAA&&ISO~X605236^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^PI^COCAA&&ISO~E40014904640^^^MEDITECH&2.16.840.1.114222.4.3.2.2.1.321.111&ISO^AN^COCAA&&ISO||BIERLY^LONNIE^^^^^L||19361023|M||2106-3^White^HL70005^W^White^L^2.5.1^5.67|4673 ELM ST^CHERRY CRK NURSING CTR^CANTERBURY^IG^80014^USA^H||^PRN^PH^^1^303^6930111|^WPN^PH^^1^303^9999999^^Business||S|NOE^Nonreligious^HL70006|||||N^Not Hispanic or Latino^HL70189^2^Not Hispanic or Latin^L^2.5.1^5.61||||||||N|||20210809085200-0600|COCAA^^ISO|337915000^Homo sapiens (organism)^SCT^Human^Human^L^1^1
 NTE|1|P|EH|RE^Remark^HL70364^C^Comment^L^2.5.1^V1
 NK1|1|SURYAN^GENARO^^^^^L|OTH^Other^HL70063^OT^OTHER RELATIONSHIP^L^2.5.1^5.67|4861 20TH AVE^^THUNDER MOUNTAIN^NV^99999^USA^H|^PRN^PH^^1^720^3013954||||||||||||||||||||||||||^WPN^PH^^1^720^3013954|4861 20TH AVE^^AURORA^CO^99999^USA^H
```

Comment:
Going to need to change the state from CO here. Noticed when running the test in staging that a receiver for the Colorado Dept of Health is also receiving the data. Because the patient state is IG and org state is CO its getting received by multiple orgs' receivers. Dunno if how bad this is but it's something I think should be proactively changed.

Reply:
Fixed this but something within the change in the valid_hl7.hl7 file made other tests failed. Took the route of making a copy of this file for just e2e tests. https://github.com/CDCgov/prime-reportstream/pull/13679/commits/29518aa9ed2e39fa79b5fd35a9803796705a0f97